### PR TITLE
Copied readme example to new examples directory and adjusted formatti…

### PR DESCRIPTION
**Pull request by bt-**
…ng to wrap lines.

I've never made a pull request, so this one is as much to understand how it works as making any useful contribution.

I just created a script file from the readme example and also wrapped the lines.  I was having a little trouble following along while having to scroll left and right.

### CODE REVIEW

File: examples/pvfit-m_ex.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# Python 3.6-7
+import numpy
+
+# This imports `pvfit_m.api` and `pvfit_m.data`.
+import pvfit_m
+
+print(f"pvfit_m version {pvfit_m.__version__}")
+
+"""
+# pvfit_m has several data classes that wrap underlying numpy.ndarray data
```
Comment (markcampanelli) [inline examples/pvfit-m_ex.py]: Remove redundant `#` .

File: examples/pvfit-m_ex.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+represting the various curves appearing in the four integrals in the
+formula for M.
+
+See, for example, equation (5) in
+https://onlinelibrary.wiley.com/doi/full/10.1002/ese3.190
+"""
+
+"""
+# pvfit_m.data has already created several useful example data objects.
+# For example, assuming one has loaded wavelength and spectral responsivity
```
Comment (markcampanelli) [inline examples/pvfit-m_ex.py]: Remove redundant `#` s in this line and previous.

**Review by markcampanelli — CHANGES_REQUESTED** [review]
Thanks for the quick contribution :). I'm using GitHub's PR review system here, and I'm requesting a couple small changes. Also, can you change the filename from `pvfit-m_ex.py` to `getting_started.py`?

I like the 80 char column limit for the example files, but note that I use a 120 char limit for source code. (Of course, I need to document all this in the README.)

I will be doing a round of further documentation shortly, and I plan to improve this getting started example (showing how to load actual numpy data and compute individual integrals for closer examination) and I will change the README to refer to running this script instead of typing it out there.

**Comment by bt-** [comment]
@markcampanelli, I made the adjustments you requested.  Let me know if you have any other comments.  I was thinking it might be helpful to have an example in an ipynb as well.

**Review by markcampanelli — APPROVED** [review]
Thanks!

**Pull request merged**